src/hotspot/share/prims/whitebox.cpp

```diff
@@ -1445,12 +1445,12 @@ WB_END
 WB_ENTRY(void, WB_FullGC(JNIEnv* env, jobject o))
   Universe::heap()->soft_ref_policy()->set_should_clear_all_soft_refs(true);
   Universe::heap()->collect(GCCause::_wb_full_gc);
-#if INCLUDE_G1GC
-  if (UseG1GC) {
-    // Needs to be cleared explicitly for G1
+#if defined(INCLUDE_G1GC) || defined(INCLUDE_SERIALGC)
```

Comment:
`INCLUDE_G1GC` can be defined as zero; it's best to treat them as bool vars. The rest look good.